x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -271,33 +320,50 @@ export const ProcessEventDot = styled(
                   textTransform: 'uppercase',
                   letterSpacing: '-0.01px',
                   backgroundColor: NamedColors.resolverBackground,
-                  lineHeight: '1.2',
+                  lineHeight: '1',
                   fontWeight: 'bold',
-                  fontSize: '.5em',
+                  fontSize: '0.8rem',
                   width: '100%',
-                  margin: '0 0 .05em 0',
+                  margin: '0',
                   textAlign: 'left',
                   padding: '0',
+                  color: NamedColors.empty,
                 }}
               >
                 {descriptionText}
               </div>
               <div
+                className={magFactorX >= 2 ? 'euiButton' : 'euiButton euiButton--small'}
                 data-test-subject="nodeLabel"
                 id={labelId}
                 style={{
                   backgroundColor: labelBackground,
-                  padding: '.15em 0',
+                  padding: '.15rem 0',
                   textAlign: 'center',
-                  maxWidth: '100%',
+                  maxWidth: '20rem',
+                  minWidth: '12rem',
+                  width: '60%',
                   overflow: 'hidden',
                   whiteSpace: 'nowrap',
                   textOverflow: 'ellipsis',
                   contain: 'content',
+                  margin: '.25rem 0 .35rem 0',
                 }}
               >
-                {eventModel.eventName(event)}
+                <span className="euiButton__content">
+                  <span className="euiButton__text">{eventModel.eventName(event)}</span>
+                </span>
               </div>
+              {magFactorX >= 2 && (
```

Comment:
it __might__ be worth considering using `scale` here instead of a value from the transform:
* The scale is a logical value from 0 to 1 that expressed how zoomed in we are. The user can control it via the range slider. 
* The value from the transform is derived from that based on some equation.

In other words, if you want to show this whenever the range slider is in the 75% position, you could compare `scale >= 0.75`

That being said, if you know that this button only makes sense when things are of a certain size on screen, and you want to base the decision on pixels, your current logic seems right.

Not necessarily recommending a change here, just wanted to mention this.

Reply:
Right now this is set to switch when there's roughly "enough space" (and it's near enough to a nice round integer). "75% level" was the name picked for the composition to help foster a general understanding of the concept of responsive nodes - not as a literal target for a transition threshold. We may want to pick better names for the responsive states when things settle down (or just adjust them to the stops they actually inhabit) - like this one might be "63%". I think that's an exercise we can take up later.